src/artm/c_interface.cc

```diff
@@ -57,19 +57,38 @@ static void set_last_error(const std::string& error) {
   last_error_->assign(error);
 }
 
-static void EnableLogging() {
+static void EnableLogging(artm::ConfigureLoggingArgs* args_ptr = nullptr) {
```

Comment:
When `EnableLogging` is called through `ArtmConfigureLogging` it is important to report back errors --- not `ARTM_SUCCESS`. I would achieve this by
1. Creating an new overload method `static void EnableLogging(const artm::ConfigureLoggingArgs& args)`; the existing method `static void EnableLogging()` is then kept unchanged.
2. remote `try` and `catch` from the overloaded method
3. in `ArtmConfigureLogging` do
```
try {
  ::artm::ConfigureLoggingArgs args;
  ParseFromArray(configure_logging_args, length, &args);
  EnableLogging(&args);
  return ARTM_SUCCESS;
} CATCH_EXCEPTIONS;
```